assets/sass/patterns/molecules/investor-logos.scss

```diff
@@ -1,63 +1,60 @@
 @import "../../definitions";
 
+$image-height: 72;
+$image-width: 185;
+$padding: $blg-space-small-in-px;
+
 .investor-logos {
-  width: 100%;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
 }
 
-.investor-logos__img_container {
+.investor-logos__item {
   border-top: 1px solid $color-text-dividers;
-  @include height(120);
+  @include padding($padding);
```

Comment:
`blg` (aka baseline grid) variables should only be used to set vertical space, not horizontal space: if the baseline grid changes, the horizontal spacing should be unaffected.